package.json

```diff
@@ -44,14 +44,14 @@
     "samples-test": "cd samples/ && npm link ../ && npm test && cd ../"
   },
   "dependencies": {
-    "@google-cloud/common": "^0.24.0",
+    "@google-cloud/common": "^0.25.1",
     "@google-cloud/projectify": "^0.3.0",
     "@google-cloud/promisify": "^0.3.0",
     "@grpc/proto-loader": "^0.3.0",
     "dot-prop": "^4.2.0",
     "duplexify": "^3.6.0",
     "extend": "^3.0.2",
-    "grpc": "^1.13.1",
+    "grpc": "~1.15.1",
```

Comment:
Is the change to `~` intentional? Can you document the rationale in the PR?

Reply:
That was an accident while I was trying something.  Fixed :)